api/v1beta1/policyautomation_types.go

```diff
@@ -14,14 +14,15 @@ type PolicyAutomationSpec struct {
 	// +kubebuilder:validation:Required
 	PolicyRef string `json:"policyRef"`
 	// Mode decides how automation is going to be triggered
-	// +kubebuilder:validation:Enum={once,disabled}
+	// +kubebuilder:validation:Enum={once,everyEvent,disabled}
 	// +kubebuilder:validation:Required
 	Mode string `json:"mode"`
 	// EventHook decides when automation is going to be triggered
 	// +kubebuilder:validation:Enum={noncompliant}
 	// +kubebuilder:validation:Required
-	EventHook   string `json:"eventHook,omitempty"`
-	RescanAfter string `json:"rescanAfter,omitempty"`
+	EventHook            string `json:"eventHook,omitempty"`
+	RescanAfter          string `json:"rescanAfter,omitempty"`
+	DelayAfterRunSeconds string `json:"delayAfterRunSeconds,omitempty"`
```

Comment:
Could you make this a `uint` or could you remove this and implement this part of the feature in a future PR?